### DISCUSSION TITLE
Copying SelfControl blocklist from one computer to another

### DISCUSSION THREAD

**Discussion by peytonstafford** [SelfControlApp/selfcontrol#694] [Q&A]
I recently bought another Mac and wanted to use SelfControl on it, but I did not want to have to rebuild the blocklist I had built over several years of use on the older machine. I found that I could move the blocklist from the older machine to the newer by using the File/Save Blocklist command from SelfControl's menu. When I double-clicked on the saved file after transferring it to the newer computer, SelfControl imported it, and it worked perfectly. It seems that many people may want to do this, and that adding a short note on how to do it would be a welcome addition to the FAQ. Thoughts? My apologies, if this topic has already been covered.

**Comment by cstigler** [comment]
Hey @peytonstafford , glad that was helpful! This is actually [already in the FAQ](https://github.com/SelfControlApp/selfcontrol/wiki/FAQ#q-can-i-save-my-blocklists-and-allowlists), but wording improvements always welcome!

**Comment by peytonstafford** [comment]
Thanks. I suggest adding a new question with its answer as follows:

*Q: I need to use SelfControl on more than one computer, or I need to move
SelfControl to a new computer. How do I move my blocklists and
allowlists so I don't have to start over with them?*

First, save your lists. Use File > Save or Command-S. This will save your
list as a file. Next, move the saved file to the other computers, just like
you would any other file. Then, install SelfControl on the other
computers, if you have not already. Last, double-click the saved file, and
its contents will be imported into SelfControl. Thereafter, you SelfControl
normally. If you want to keep all your SelfControl installations running
the same lists, you will need to repeat the process of saving, moving, and
double-clicking the file each time you go from one machine to another.

Peyton Stafford | Voice +1 503.720.7149 | www.peytonstafford.com |  See my
forthcoming book from ALA Editions: No Shelf Required 3: The New Era for
E-Books and Digital Content
<https://www.alastore.ala.org/content/no-shelf-required-3-new-era-e-books-and-digital-content>
.
